src/main/java/net/minecraftforge/fluids/FluidType.java

```diff
@@ -1151,5 +1170,30 @@ public Properties rarity(Rarity rarity)
             this.rarity = rarity;
             return this;
         }
+
+        /**
+         * Allows this fluid to drip from Pointed Dripstones and fill cauldrons below.
+         *
+         * @param chance the chance that the cauldron below will be filled every time the Pointed Dripstone is randomly ticked
+         * @param dripParticle the particle that spawns randomly from the tip of the Pointed Dripstone when this fluid is above it
+         * @param cauldron the block the Pointed Dripstone should replace an empty cauldron with when it successfully tries to fill the cauldron
+         * @return the property holder instance
+         */
+        public Properties addDripstoneDripping(float chance, ParticleOptions dripParticle, Block cauldron)
```

Comment:
You can still have an overload that takes in the fill sound and then just redirect that to the sound action.